sunbird-es-utils/src/main/java/org/sunbird/common/inf/ElasticSearchUtil.java

```diff
@@ -0,0 +1,114 @@
+package org.sunbird.common.inf;
+
+import java.util.List;
+import java.util.Map;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ProjectUtil.EsType;
+import org.sunbird.dto.SearchDTO;
+import scala.concurrent.Future;
+
+public interface ElasticSearchUtil {
```

Comment:
ElasticSearchUtil need to be rename as ElasticService 
as this is not a util class.

Reply:
Done